pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1105,6 +1105,51 @@ func parseOneNetworkXObjectConfig(ctx *getconfigContext, netEnt *zconfig.Network
 		config.Proxy = &proxyConfig
 	}
 
+	// wireless property configuration
+	netWireless := netEnt.GetWireless()
+	if netWireless != nil {
+		log.Infof("parseOneNetworkXObjectConfig: Wireless of network present in %s, config %v", netEnt.Id, netWireless)
```

Comment:
Make sure no secrets get logged here..

Reply:
right. no secrets is never in clear text in structures here.